docs/reference/commandline/swarm_init.md

```diff
@@ -22,6 +22,7 @@ Initialize a swarm
 
 Options:
       --advertise-addr value            Advertised address (format: <ip|interface>[:port])
+      --autolock                        Enable or disable manager autolocking (requiring an unlock key to start a stopped manager)
```

Comment:
@thaJeztah: Can you suggest a phrasing for this? `--autolock` or `--autolock=true` will enable it, and `--autolock=false` will disable it (in cases where it was previously enabled).